conf/lookup/BCD-Port.csv

```diff
@@ -11479,7 +11478,7 @@ PortId,CountryCode,PortType,PortCode,KeyWords,Name,Name-cy,Priority,Region,Locat
 32563,UK,3,MRY,,Maryport Seaport - MRY,Maryport Seaport - MRY,,1,NULL,1,
 32578,UK,3,MEB,,Menai Bridge Seaport - MEB,Menai Bridge Seaport - MEB,,1,NULL,1,
 32584,UK,3,MTH,,Methil Seaport - MTH,Methil Seaport - MTH,,1,NULL,1,
-32590,UK,3,MID,,Middlesborough Seaport - MID,Middlesborough Seaport - MID,,1,NULL,1,
+32590,UK,3,MID,Teesport,Middlesborough Seaport - MID,Middlesborough Seaport - MID,,1,NULL,1,
 32592,UK,3,MDW,,Middlewich Seaport - MDW,Middlewich Seaport - MDW,,1,NULL,1,
 32595,UK,2,MLL,,Mildenhall Airport - MLL,Mildenhall Airport - MLL,,1,NULL,1,
 32596,UK,3,MLF,,Milford Haven Seaport - MLF,Milford Haven Seaport - MLF,,1,NULL,1,

```

Comment:
Line is now part of:

200066,UK,3,MIL,Pembroke Olis Epu Ferry,Milford - MIL,Milford - MIL,,1,NULL,0,

This should be deleted

Reply:
Keep both